data/boot/theme.file_list

```diff
@@ -5,7 +5,7 @@ if arch eq 'i386' || arch eq 'x86_64'
   gfxboot-branding-<gfxboot_theme>:
     /
     e cp -a etc/bootsplash/themes/<gfxboot_theme>/cdrom/* loader
-    e gfxboot --config-file=loader/gfxboot.cfg --change-config install::dud.url=http://download.opensuse.org/update/openSUSE/driverupdate
+    # e gfxboot --config-file=loader/gfxboot.cfg --change-config install::dud.url=http://download.opensuse.org/update/openSUSE/driverupdate
```

Comment:
Is there any reason to keep it commented?

Reply:
in case we want to re-add in the future people can see how it's done

Reply:
Well, it is also saved in the git history, but ok, LGTM.